commcare-cloud/commcare_cloud/commands/ansible/downtime.py

```diff
@@ -0,0 +1,183 @@
+# coding: utf-8
+import os
+import time
+from datetime import datetime
+
+import datadog
+import yaml
+from clint.textui import puts, colored, indent
+from memoized import memoized
+
+from commcare_cloud.cli_utils import ask, ask_option
+from commcare_cloud.commands.ansible.helpers import AnsibleContext
+from commcare_cloud.commands.ansible.run_module import run_ansible_module
+from commcare_cloud.commands.command_base import CommandBase
+from commcare_cloud.environment.main import get_environment
+
+HQ_PROCESSES_SCOPE = 'webworkers,celery,pillowtop,touchforms,formplayer,proxy'
+
+
+class Downtime(CommandBase):
+    command = 'downtime'
+    help = (
+        'Manage downtime for the selected environment.'
+    )
+
+    def make_parser(self):
+        self.parser.add_argument('action', choices=('start', 'check', 'end'))
+        self.parser.add_argument('-m', '--message', help='Optional message to set on Datadog')
+
+    def run(self, args, unknown_args):
+        environment = get_environment(args.env_name)
+        environment.create_generated_yml()
+        ansible_context = AnsibleContext(args)
+
+        if args.action == 'start':
+            start_downtime(environment, ansible_context, args)
+
+        if args.action == 'check':
```

Comment:
When might you use check?

Reply:
I had thought it could be useful but can't really think of a good use case right now so I've removed it.